index.jsx

```diff
@@ -117,6 +117,8 @@ const AnimatedNumber = ({
   );
 };
 
-const Enhanced = React.memo(AnimatedNumber)
+const Enhanced = React.memo(AnimatedNumber, (prevProps, nextProps) => {
+    return prevProps.animateToNumber === nextProps.animateToNumber
```

Comment:
Since **fontstyle** can also be a prop, I think it would be better to add a dependency for it as well.